README.md

```diff
@@ -48,22 +47,32 @@ var TryStardust = React.createClass({
 module.exports = TryStardust;
 ```
 
-## Develop
+## Installation
+Start by cloning this repo and installing dependencies:
+
+    git clone git@github.com:TechnologyAdvice/stardust.git stardust && cd $_
+    npm install               # or simply npm i
+
+## Usage
+Once dependencies are installed, start the application with:
+
+    npm start
```

Comment:
I prefer to use a standard README document outline I see in most libraries so it's easier for users to grok our documentation. But I'll change this back so we can move forward.